src/main/java/world/bentobox/bentobox/hooks/LangUtilsHook.java

```diff
@@ -0,0 +1,567 @@
+    }
+
+    /**
+     * Return the name of the enchantment.
+     *
+     * @param enchant The enchantment.
+     * @param user    The User's locale will be used for translation.
+     * @return The translated enchant name.
+     */
+    @SuppressWarnings("unused")
+    public static String getEnchantName(Enchantment enchant, User user) {
+        return hooked
+                ? LanguageHelper.getEnchantmentName(enchant, getUserLocale(user))
+                : enchant.getKey().getKey();
+    }
+
+    /**
+     * Return the enchantment level indicated by Roman numerals.
+     * Can only get Roman numerals within 10.
+     *
+     * @param level The enchantment level.
+     * @param user  The user's language will be used for translation.
+     * @return The converted enchantment level.
+     */
+    @SuppressWarnings("unused")
+    public static String getEnchantLevelName(int level, User user) {
+        return hooked
+                ? LanguageHelper.getEnchantmentLevelName(level, getUserLocale(user))
+                : String.valueOf(level);
+    }
+
+    /**
+     * Translate the name of the potion.
+     *
+     * @param potionType The type of the potion.
+     * @param user       The user's language will be used for translation.
+     * @return Translated potion name.
+     */
+    @SuppressWarnings("unused")
+    public static String getPotionTypeName(PotionType potionType, User user) {
+        if (hooked) {
+            return LanguageHelper.getPotionName(potionType, getUserLocale(user));
+        }
+        switch (potionType) {
+            case UNCRAFTABLE:     return "Uncraftable Potion";
+            case WATER:           return "Water Bottle";
+            case MUNDANE:         return "Mundane Potion";
+            case THICK:           return "Thick Potion";
+            case AWKWARD:         return "Awkward Potion";
+            case NIGHT_VISION:    return "Potion of Night Vision";
+            case INVISIBILITY:    return "Potion of Invisibility";
+            case JUMP:            return "Potion of Leaping";
+            case FIRE_RESISTANCE: return "Potion of Fire Resistance";
+            case SPEED:           return "Potion of Swiftness";
+            case SLOWNESS:        return "Potion of Slowness";
+            case WATER_BREATHING: return "Potion of Water Breathing";
+            case INSTANT_HEAL:    return "Potion of Healing";
+            case INSTANT_DAMAGE:  return "Potion of Harming";
+            case POISON:          return "Potion of Poison";
+            case REGEN:           return "Potion of Regeneration";
+            case STRENGTH:        return "Potion of Strength";
+            case WEAKNESS:        return "Potion of Weakness";
+            case LUCK:            return "Potion of Luck";
+            case TURTLE_MASTER:   return "Potion of the Turtle Master";
+            case SLOW_FALLING:    return "Potion of Slow Falling";
+            default:
```

Comment:
I do not thing it is a good idea to throw exception.
If something went wrong, it should return untranslated text and not crash.

Reply:
You're right. I'll revise it now.